game-to-flatpak

```diff
@@ -49,6 +52,15 @@ for k, v in ipairs(arg) do
 	end
 end
 
+if not options.repo_dir then
+    options.repo_dir = 'repo'
+elseif not options.keep_repo_dir then
+    -- Adding it to your local repo and also making a bundle should not delete
+    -- the whole local repo.
+    print('Repo directory defined, assuming --keep-repo.')
+    options.keep_repo_dir = true
```

Comment:
I do actually set `options.repo_dir` to 'repo', but only if it is empty in the if block right above that one. These two lines are only executed if it is set and `options.keep_repo_dir` is not set. Did I miss something?